spec/domain/queries/find_series_spec.rb

```diff
@@ -1,108 +1,93 @@
-      metric1 = create :metric, edition: edition, date: '2018-5-13'
-      metric2 = create :metric, edition: edition, date: '2018-5-14'
-      create :metric, edition: edition, date: '2018-5-24'
-      series = described_class.new.between(from: '2018-5-13', to: '2018-5-14').run
-
-      expect(series).to_not be_nil
-      expect(series.size).to eq(2)
-      expect(series).to match_array([metric1, metric2])
+      create :metric, edition: edition, date: '2018-5-13', pviews: 1
+      create :metric, edition: edition, date: '2018-5-14', pviews: 2
+      create :metric, edition: edition, date: '2018-5-24', pviews: 3
+
+      series = described_class.new.by_metrics(%w(pviews)).between(from: '2018-5-13', to: '2018-5-14').run
+
+      expect(series.first.time_series.length).to eq(2)
+      expect(series.first.time_series).to eq([
+        { date: '2018-05-13', value: 1 },
+        { date: '2018-05-14', value: 2 },
+      ])
     end
   end
 
   context "by_metric" do
     it "returns a series of metrics filtered by the passed in metric" do
-      today = Date.today
-      tomorrow = today + 1
-
-      edition1 = create :edition, base_path: '/path/1', date: today, facts: { words: 10_000 }
-      metric1 = create :metric, date: today, edition: edition1
-      edition2 = create :edition, base_path: '/path/2', date: tomorrow, facts: { words: 20_000 }
-      metric2 = create :metric, date: tomorrow, edition: edition2
-      metric3 = create :metric, date: tomorrow
-
-      series = described_class.new.run
+      create :metric, date: Date.today, pviews: 1
 
-      expect(series).to match_array([metric1, metric2, metric3])
+      expect(described_class.new.by_metrics(%w(pviews)).run.length).to eq(1)
     end
   end
 
   context "by_organisation" do
     it "returns a series of metrics filtered by organisation" do
-      day0 = Date.new(2018, 1, 12)
-      day1 = Date.new(2018, 1, 13)
-      day2 = Date.new(2018, 1, 14)
-      edition1 = create :edition, date: day0, organisation_id: 'org-1'
-      edition2 = create :edition, date: day1, organisation_id: 'org-2'
+      edition1 = create :edition, date: '2018-1-12', organisation_id: 'org-1'
+      edition2 = create :edition, date: '2018-1-13', organisation_id: 'org-2'
 
-      metric1 = create(:metric, edition: edition1, date: day0)
-      metric2 = create(:metric, edition: edition1, date: day1)
-      create(:metric, edition: edition2, date: day2)
-
-      series = described_class.new.by_organisation_id('org-1').run
-
-      expect(series).to match_array([metric1, metric2])
-    end
+      create :metric, edition: edition1, date: '2018-1-12', pviews: 1
+      create :metric, edition: edition1, date: '2018-1-13', pviews: 2
+      create :metric, edition: edition2, date: '2018-1-14', pviews: 1
 
-    it 'ignores parameters when blank' do
-      metric = create(:metric, date: Date.today)
+      series = described_class.new.by_metrics(%w(pviews)).by_organisation_id('org-1').run
```

Comment:
Do we need to call `by_metrics`?

Reply:
Addressed in https://github.com/alphagov/content-performance-manager/pull/959/commits/55942bf20560782583f88c08b5c7e5e685edb877